src/Socialite/EveOnline/Provider.php

```diff
@@ -166,7 +166,7 @@ private function validateJwtToken(string $access_token): array
         $jws = Load::jws($access_token)
             ->algs(['RS256', 'ES256', 'HS256'])
             ->exp()
-            ->iss('login.eveonline.com')
+            // ->iss('login.eveonline.com') // Disabled temporarily to account for CCP iss change. SeAT v5 will re-enable with proper handling.
```

Comment:
```suggestion
            ->claim('iss', new \Jose\Component\Checker\IssuerChecker(['https://login.eveonline.com', 'login.eveonline.com'], true))
```

iss method is an helper to call IssuerChecker https://github.com/web-token/jwt-easy/blob/9aa087db5995e02ccf157716943232fca47a9ab2/AbstractLoader.php#L91
but you can always relay on base method which is `claim` https://github.com/web-token/jwt-easy/blob/9aa087db5995e02ccf157716943232fca47a9ab2/AbstractLoader.php#L109

notice you will need the same changes applied to eseye 2.x